### PR TITLE
Add token details

### PR DESCRIPTION
**Pull request by netanelC**
This PR adds the JWT parser and makes the names shorter

### CODE REVIEW

**Review by EllaMartirosyan — CHANGES_REQUESTED** [review]
This branch is not synced with the master, please merge it, we also need this changes

**Review by EllaMartirosyan — APPROVED** [review]

**Review by alebinson — APPROVED** [review]
gr8!

File: helm/values.yaml [hunk 1]
```diff
@@ -74,6 +74,7 @@ s3:
   style: path
   debug: true
   nginxWorkerProcesses: 4
+  noCache: true
```
Comment (alebinson) [inline helm/values.yaml]: think, by default should be **false**

**Review by alebinson — CHANGES_REQUESTED** [review]

File: helm/values.yaml (hunk 1 above)
Comment (EllaMartirosyan) [reply helm/values.yaml]: I know, I'm still testing this feature...

**Review by netanelC — COMMENTED** [review]
If configmap conf is empty, don't point to that configmap. Instead, you can do an if statement to not point to the empty configmap. @EllaMartirosyan

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**